### PR TITLE
Refactored Start.vue

### LINKED ISSUE #496: Refactor Start.vue
### Description
Refactor Start.vue and related components to the Composition API for future-proofing. 

### Acceptance Criteria
- [ ] Refactor Start.vue
- [ ] Refactor LoadingIcon.vue

### Solution Proposal
- Just Do It

### PR DESCRIPTION
**Pull request by te4-alexander-kjellberg**
## ⚙️ What issue did you implement or fix?
Fixes #496 

## 📜 Summary
- Refactored Start.vue to Composition API. 
- Realised that LoadingIcon is never used => removed corresponding imports and references. 
- Re-wrote some descriptions. 

## 📝 Type of change

- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [x] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Adding tests wouldn't hurt...

### CODE REVIEW

**Review by Te4-Mahdi-Alizadah — APPROVED** [review]
That seems good.

File: frontend/src/views/Start.vue [hunk 1]
```diff
@@ -87,40 +87,73 @@
...
+//   // there is a relation between loans and users the constructor instantiates the class
+//   // by fetching all the information from the two modules
+
+//   public created(): void {
+//     if(UsersModule.currentUserID){
+//       UsersModule.fetchAll();
+//       LoansModule.fetchAll();
+//     }
+//   }
+// }
```
Comment (itggot-linus-styren) [inline frontend/src/views/Start.vue:156]: Please consider removing this ;)

See: https://kentcdodds.com/blog/please-dont-commit-commented-out-code/

**Review by te4-andre-skvarc — CHANGES_REQUESTED** [review]
Solve the code that Linus mentioned! And it will be ready!

**Review by TE4-Filip-Liljenberg — APPROVED** [review]
Great succes!

**Review by te4-andre-skvarc — APPROVED** [review]
Now it's looking much better, approved! 👍

**Pull request merged**